include/net/socket.h

```diff
@@ -128,6 +128,12 @@ struct zsock_pollfd {
  *  the TLS handshake.
```

Comment:
Could you fix the commit subject, we are actually not changing anything in mbedtls module so better subject would be
`net: socket: dtls: add sockopt to set & get handshake timeout`

Reply:
I have done this